files/import_eoni_from_s3.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+set -x
+
+
+LATEST_FILE=$(aws s3 ls s3://eoni-data.wheredoivote.co.uk/production_data/ | sort | tail -n1 | rev | cut -d' ' -f1 | rev)
+SRCDIR='/tmp/eoni_production_data'
+mkdir -p $SRCDIR
```

Comment:
Could you `rm -rf $SRCDIR && mkdir -p $SRCDIR` here? This will empty the folder, meaning we don't max the hard drive out on servers that hang about for months